### PR TITLE
Fixes #4963: Makes color of clipboardIcon black

### LINKED ISSUE #4963: Clipboard suggestion icon is white
It's very difficult to see the default suggestion in Sample Browser because the icon is white. 

image

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-4219)

### PR DESCRIPTION
**Pull request by sblatz**
---

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- [ ] **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

**Comment by pocmo** [comment]
Oh, I see this is for sample browser. I was confused since afaik Fenix doesn't use the clipboard suggestion provider at the moment? (Which makes me sad!)

File: components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/provider/ClipboardSuggestionProvider.kt [hunk 1]
```diff
@@ -35,19 +36,23 @@ class ClipboardSuggestionProvider(
...
+        if ((requireEmptyText && text.isEmpty()) || !requireEmptyText) createClipboardSuggestion() else emptyList()
 
     private fun createClipboardSuggestion(): List<AwesomeBar.Suggestion> {
         val url = getTextFromClipboard(clipboardManager)?.let {
             findUrl(it)
         } ?: return emptyList()
 
+        val clipboardIcon = context.getDrawable(R.drawable.mozac_ic_search)?.apply {
+            setTint(ContextCompat.getColor(context, android.R.color.black))
+        }
```
[the remark this answers is not in the document]
Comment (sblatz) [reply components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/provider/ClipboardSuggestionProvider.kt]: Yes, this is just for sample browsers *default icon*, so I think filling it with black seems reasonable?

[the remark this answers is not in the document]
Comment (pocmo) [reply components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/provider/ClipboardSuggestionProvider.kt]: You will most definitely break Reference Browser, which is dark, by always picking black. :)

**Comment by pocmo** [comment]
Closing for now: I think we should not hard-code the color since that can break apps and themes. It can be customized via `mozac_ui_icons_fill` but that may not be enough if this is the only icon that should be colored. Not sure if it is worth adding an API for that at the moment (but feel free to open a PR if you want to).

**Pull request closed without merging**